infra/Makefile

```diff
@@ -0,0 +1,7 @@
+ENVIRONMENT?=staging
+
```

Comment:
When I tried to `make init`, I noticed my `gcloud` application default credentials expired lol. Let's add another recipe for setting that up.

```
gcloud-login:
	cd $(ENVIRONMENT)
	gcloud auth application-default login
```

Reply:
https://github.com/bettersg/call-home/pull/87/commits/d082e99765c55115663117c99cc9703b30ad73e2